### PR TITLE
add seeds to the project

### PR DESCRIPTION
**Pull request by bedreski**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,4 +5,15 @@
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+  # Character.create(name: "Luke", movie: movies.first)
+  positions = Position.create([{ name: "Gerente" }, { name: "CEO" }, { name: "Assistente" }])
+  employees = Employee.create([{ name: "Jessie" }, { name: "Tony" }, { name: "Marie" }])
+  departments = Department.create([{ name: "Finanças", manager_id: 1 }, { name: "Design de Produto", manager_id: 2} ])
```
Comment (EduardoSCosta) [inline db/seeds.rb:11]: vc pode usar os registros q vc ja criou:
```suggestion
  departments = Department.create([{ name: "Finanças", manager_id: employees[0] }, { name: "Design de Produto", manager_id: employees[1]} ])
```

File: db/seeds.rb [hunk 2]
```diff
@@ -5,4 +5,15 @@
...
+  projects = Project.create([
+                      { name: "Amazing Project", starts_on: 3.days.ago }, 
+                      { name: "Trouble Project", starts_on: 3.months.ago, budget: 1000, spent: 1500 },
+                      { name: "Ok Project", starts_on: Date.today, budget: 1000, spent: 900 },
+                    ])
+  assignments = Assignment.create([{ employee_id: 1, project_id: 2, starts_on: Date.today }, { employee_id: 2, project_id: 3, starts_on: Date.today }])
```
Comment (EduardoSCosta) [inline db/seeds.rb:17]: o mesmo aqui

File: db/seeds.rb (hunk 1 above)
Comment (bedreski) [reply db/seeds.rb:11]: Yes, sur

File: db/seeds.rb (hunk 2 above)
Comment (bedreski) [reply db/seeds.rb:17]: Yes, sur

**Review by EduardoSCosta — APPROVED** [review]
LGTM

**Review by HeitorMC — COMMENTED** [review]
As a suggestion for the following PRs, you should always add a description of your pr. This helps the reviewer during the process. Also, improve the PR title, something like:

`Feature: Project seeds`

**Comment by HeitorMC** [comment]
I liked your commits, but you can improve them even more. Please, check this [link](http://karma-runner.github.io/1.0/dev/git-commit-msg.html) later on :smile:

**Review by HeitorMC — APPROVED** [review]
Overall, LGTM :+1:

**Pull request merged**